skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -16,3 +16,6 @@
 
 skaffold.run.config.general.name=Skaffold
 skaffold.run.config.general.description=Runs Skaffold
+
+skaffold.run.config.single.run=Skaffold Single Run
```

Comment:
these are the UI visible names of the run configurations (as shown in your screenshot) right?

Maybe make the name of the variable more clear to indicate that. Maybe just adding a `.name` to the end.